openproblems/tasks/_batch_integration/batch_integration_graph/methods/_utils.py

```diff
@@ -0,0 +1,21 @@
+def hvg_batch(adata, batch_key, target_genes, adataOut):
+    from scib.preprocessing import hvg_batch
+
+    if adata.n_vars < 2000:
+        return adata
+    else:
+        hvg_batch(
+            adata,
+            batch_key=batch_key,
+            target_genes=target_genes,
+            flavor="cell_ranger",
+            adataOut=adataOut,
+        )
```

Comment:
I guess this code will have to be duplicated between graph embedding and feature output tasks? Any way to make this a util function outside batch_integration_graph?

Reply:
you dont' have a return statement in the `else:` block.

Reply:
I think this would probably require an api change? Not sure tbh